packages/calypso-products/src/translations.tsx

```diff
@@ -566,7 +566,9 @@ export const getJetpackProductsShortDescriptions = (): Record< string, Translate
 	const backupRealtimeShortDescription = translate(
 		'Real-time cloud backups with one-click restores.'
 	);
-	const backupShortDescription = translate( 'Real-time cloud backups with one-click restores.' );
+	const backupShortDescription = translate(
+		'Real-time cloud backups with one-click restores. Starts with 10GB.'
```

Comment:
I think it would be good to pass '10GB' as an argument to the translate string, I doubt that's going to change soon but it's still best practices to do that. Another example of this is here: https://github.com/Automattic/wp-calypso/blob/trunk/client/jetpack-cloud/sections/comparison/table/useComparisonData.tsx#L54

Reply:
Done 👍  Let me know if you see anything off.